eng/tools/spec-gen-sdk-runner/src/utils.ts

```diff
@@ -409,7 +409,7 @@ export function groupPathsByService(
     }
 
     const info = serviceMap.get(serviceName)!;
-    if (folderPath.endsWith(".Management")) {
+    if (folderPath.endsWith(".Management") || folderPath.includes("resource-manager")) {
```

Comment:
@raych1: I assume you still want this merged to main as part of this PR, even though you already merged #35108?

Reply:
It's ok to merge it as it won't introduce bad impact.